README.md

```diff
@@ -242,7 +242,7 @@ instead of changing your local cactbot files.
 
 To install npm and start Webpack, follow these steps:
 
-1. Install [nodejs and npm](https://nodejs.org/en/download/)
+1. Install [nodejs 18.17.1 and npm 9.6.7](https://nodejs.org/en/download/)
```

Comment:
```suggestion
1. Install [nodejs 18.17.1 and npm 9.6.7](https://nodejs.org/en/download/)
```

This feels like the same issue here as above. Re: nodejs, I feel like cactbot should always work against the LTS version of nodejs and if it doesn't then it's a bug to be fixed. As soon as 18.17.1 is not the most current LTS version, then it's going to be much harder to find as well and folks will have to dig around in versions to get to it.

I'd rather make a note that says "cactbot should always work with the latest LTS and file an issue if it doesn't".

Reply:
Fair point, that is definitely a more sensible solution.